nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-ui/src/main/webapp/js/nf/canvas/controllers/nf-ng-canvas-flow-status-controller.js

```diff
@@ -400,6 +406,562 @@
+                                version: ruleInfo.bundle.version
+                            })).done(function () {});
+                            $("#rule-menu").hide();
+                            unbindRuleMenuHandling();
+                        }
+
+                        function closeRuleWindow(e) {
+                            if ($(e.target).parents("#rule-menu").length === 0) {
+                                $("#rule-menu").hide();
+                                unbindRuleMenuHandling();
+                            }
+                        }
+
+                        function openRuleDetailsDialog() {
+                            $('#rule-menu').hide();
+                            nfSettings.showSettings().done(function() {
+                                nfSettings.selectFlowAnalysisRule(ruleInfo.id);
+                            });
+                            unbindRuleMenuHandling();
+                        }
+
+                        function unbindRuleMenuHandling() {
+                            $('#rule-menu-edit-rule').off("click");
+                            $('#rule-menu-view-documentation').off("click");
+                            $(document).unbind('click', closeRuleWindow);
+                        }
+
+                    });
+                },
+
+                /**
+                 * Set event bindings for violation menus
+                 */
+                setViolationMenuHandling: function(rules) {
+                    $('.violation-menu-btn').click(function(event) {
+                        // stop event from immediately bubbling up to document and triggering closeViolationWindow
+                        event.stopPropagation();
+                        var violationInfo = $(this).data('violationInfo');
+                        $('#rule-menu').hide();
+                        $('#violation-menu').show();
+                        $('#violation-menu').position({
+                            my: "left top",
+                            at: "left top",
+                            of: event
+                        });
+
+                        // violation menu bindings
+                        if (violationInfo.subjectPermissionDto.canRead) {
+                            $('#violation-menu-more-info').removeClass('disabled');
+                            $('#violation-menu-more-info .violation-menu-option-icon').removeClass('disabled');
+                            $('#violation-menu-more-info').on( "click", openRuleViolationMoreInfoDialog);
+                        } else {
+                            $('#violation-menu-more-info').addClass('disabled');
+                            $('#violation-menu-more-info .violation-menu-option-icon').addClass('disabled');
+                        }
+                        
+                        var isProcessor = violationInfo.subjectComponentType === 'PROCESSOR';
+                        if (violationInfo.groupId && isProcessor) {
```

Comment:
Thanks for updating the `isRootGroup` logic here but I'm still unsure of this condition. A Processor will always have a parent group. Other extension types will not have a parent group or will conditionally have a parent group. If the intent is to only support Go To for Processors then I think we only need to check if it's a Processor.